tests/integration/test_basic.py

```diff
@@ -246,6 +246,7 @@ def test_publish_mk8s_mixed_status(self, mixed_status_tests_for_mk8s):
             pytest.helpers.assert_qtest_property(test_runs[0], 'Status', test_run_status_exp)
 
 
+@pytest.mark.skip(reason="The mechanism for specifying module hiararchy has changed")
```

Comment:
Create a tech debt ticket for updating/removing the test and reference ticket # in this message.

Reply:
I created https://rpc-openstack.atlassian.net/browse/ASC-1636
It's possible that the test that @mwbutcher is working on will cover these cases.